From: Kir Kolyshkin <kir@openvz.org>
Date: Fri, 3 Sep 2010 17:47:12 +0000 (+0400)
Subject: Fix init.d Makefile
X-Git-Url: http://git.openvz.org/?p=vzctl;a=commitdiff_plain;h=c73f30939788f2d7404be1b5aa5e58a455105445

Fix init.d Makefile

Apparently I forgot to add new prerequisites to install targets.
Also, vzfunctions should be installed without executable bits set.
This is an addition to commit 4cc6257616e1a725650d6b026a42d17cd1c3cce8.

Signed-off-by: Kir Kolyshkin <kir@openvz.org>
---

diff --git a/etc/init.d/Makefile.am b/etc/init.d/Makefile.am
index 2181583..60966e8 100644
--- a/etc/init.d/Makefile.am
+++ b/etc/init.d/Makefile.am
@@ -23,9 +23,9 @@ CLEANFILES = $(INITSCRIPTS)
 
 include $(top_srcdir)/pathsubst.am
 
-install-redhat: vz-redhat
+install-redhat: vzfunctions vz-redhat vzeventd-redhat
 	$(mkinstalldirs) $(DESTDIR)$(initddir)
-	$(INSTALL_SCRIPT) vzfunctions $(DESTDIR)$(initddir)/vzfunctions
+	$(INSTALL_DATA) vzfunctions $(DESTDIR)$(initddir)/vzfunctions
 	$(INSTALL_SCRIPT) vz-redhat $(DESTDIR)$(initddir)/vz
 	$(INSTALL_SCRIPT) vzeventd-redhat $(DESTDIR)$(initddir)/vzeventd
 
@@ -33,7 +33,7 @@ install-suse: install-redhat
 
 install-debian: install-redhat
 
-install-gentoo: vz-gentoo
+install-gentoo: vz-gentoo vzeventd-gentoo
 	$(mkinstalldirs) $(DESTDIR)$(initddir)
 	$(INSTALL_SCRIPT) vz-gentoo $(DESTDIR)$(initddir)/vz
 	$(INSTALL_SCRIPT) vzeventd-gentoo $(DESTDIR)$(initddir)/vzeventd
